tests/arc/tasyncleak.nim

```diff
@@ -1,5 +1,5 @@
 discard """
-  outputsub: "(allocCount: 4302, deallocCount: 4300)"
```

Comment:
yeah, with my current PR
it is `(allocCount: 4302, deallocCount: 4300)` on Linux, and `(allocCount: 4303, deallocCount: 4301)` on Windows.